scripts/test-macos-enterprise-packaging.sh

```
@@ -143,6 +148,33 @@ rm -f -- "${probe_marker}/existing-state"
 rmdir -- "$probe_marker"
 probe_owned=false
 
+if sudo -n "$installer" \
+    --binary "$binary" \
+    --config "$config_source" \
+    --manifest "$manifest_source" \
+    --no-start >"${fixture}/untrusted-source.stdout" 2>"${fixture}/untrusted-source.stderr"; then
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)